schema/schemaspec/extension.go

```diff
@@ -94,6 +108,30 @@ func (r *Resource) As(target interface{}) error {
 				return err
 			}
 			delete(existingAttrs, attr.K)
+		case ft.isInterfaceSlice:
```

Comment:
why isInterfaceSlice is a struct attribute by isResourceSlice() requires a func?